lib/engine/game/g_18_esp/corporation.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G18ESP
+      class Corporation < Engine::Corporation
+        attr_reader :destination, :goals_reached_counter
+        attr_accessor :destination_connected, :ran_offboard, :ran_harbor_mine, :taken_over_minor, :moved_token, :full_cap
+
```

Comment:
Corporations start the game with 1 home toke, 1 token, and the rest are unusable.
Corps gain money and tokens as they complete different goals.
All that info is tracked in the corporation subclass